environment.yml

```diff
@@ -0,0 +1,168 @@
+  - libstdcxx-ng=11.2.0=h1234567_1
+  - libtiff=4.5.1=h6a678d5_0
+  - libuuid=1.41.5=h5eee18b_0
+  - libwebp=1.2.4=h11a3e52_1
+  - libwebp-base=1.2.4=h5eee18b_1
+  - libxcb=1.15=h7f8727e_0
+  - libxkbcommon=1.0.1=h5eee18b_1
+  - libxml2=2.10.4=hcbfbd50_0
+  - libxslt=1.1.37=h2085143_0
+  - lz4-c=1.9.4=h6a678d5_0
+  - markdown=3.4.1=py310h06a4308_0
+  - matplotlib=3.7.2=py310h06a4308_0
+  - matplotlib-base=3.7.2=py310h1128e8f_0
+  - mkl=2023.1.0=h213fc3f_46343
+  - mkl-service=2.4.0=py310h5eee18b_1
+  - mkl_fft=1.3.6=py310h1128e8f_1
+  - mkl_random=1.2.2=py310h1128e8f_1
+  - multidict=6.0.2=py310h5eee18b_0
+  - munkres=1.1.4=py_0
+  - mysql=5.7.24=h721c034_2
+  - ncurses=6.4=h6a678d5_0
+  - networkx=3.1=py310h06a4308_0
+  - nspr=4.35=h6a678d5_0
+  - nss=3.89.1=h6a678d5_0
+  - numexpr=2.8.4=py310h85018f9_1
+  - numpy=1.25.2=py310h5f9d8c6_0
+  - numpy-base=1.25.2=py310hb5e798b_0
+  - oauthlib=3.2.2=py310h06a4308_0
+  - openssl=3.0.10=h7f8727e_2
+  - packaging=23.1=py310h06a4308_0
+  - pandas=2.0.3=py310h1128e8f_0
+  - pcre=8.45=h295c915_0
+  - pillow=9.4.0=py310h6a678d5_0
+  - pip=23.2.1=py310h06a4308_0
+  - ply=3.11=py310h06a4308_0
+  - protobuf=3.20.3=py310h6a678d5_0
+  - pyasn1=0.4.8=pyhd3eb1b0_0
+  - pyasn1-modules=0.2.8=py_0
+  - pycparser=2.21=pyhd3eb1b0_0
+  - pyjwt=2.4.0=py310h06a4308_0
+  - pyopenssl=23.2.0=py310h06a4308_0
+  - pyqt=5.15.7=py310h6a678d5_1
+  - pysocks=1.7.1=py310h06a4308_0
+  - python=3.10.13=h955ad1f_0
+  - python-dateutil=2.8.2=pyhd3eb1b0_0
+  - python-tzdata=2023.3=pyhd3eb1b0_0
+  - pytz=2022.7=py310h06a4308_0
+  - qt-main=5.15.2=h7358343_9
+  - qt-webengine=5.15.9=h9ab4d14_7
+  - qtwebkit=5.212=h3fafdc1_5
+  - re2=2022.04.01=h295c915_0
+  - readline=8.2=h5eee18b_0
+  - requests=2.31.0=py310h06a4308_0
+  - requests-oauthlib=1.3.0=py_0
+  - rsa=4.7.2=pyhd3eb1b0_1
+  - setuptools=68.0.0=py310h06a4308_0
+  - sip=6.6.2=py310h6a678d5_0
+  - six=1.16.0=pyhd3eb1b0_1
+  - sqlite=3.41.2=h5eee18b_0
+  - tbb=2021.8.0=hdb19cb5_0
+  - tensorboard=2.12.1=py310h06a4308_0
+  - tensorboard-data-server=0.7.0=py310h52d8a92_0
+  - tensorboard-plugin-wit=1.8.1=py310h06a4308_0
+  - tk=8.6.12=h1ccaba5_0
+  - toml=0.10.2=pyhd3eb1b0_0
+  - tornado=6.3.2=py310h5eee18b_0
+  - tzdata=2023c=h04d1e81_0
+  - werkzeug=2.2.3=py310h06a4308_0
+  - wheel=0.38.4=py310h06a4308_0
+  - xz=5.4.2=h5eee18b_0
+  - yarl=1.8.1=py310h5eee18b_0
+  - zlib=1.2.13=h5eee18b_0
+  - zstd=1.5.5=hc292b87_0
+  - pip:
+      - charset-normalizer==3.2.0
+      - cmake==3.27.4.1
+      - filelock==3.12.3
+      - jinja2==3.1.2
+      - joblib==1.3.2
+      - lit==16.0.6
+      - markupsafe==2.1.3
+      - mpmath==1.3.0
+      - psutil==5.9.5
+      - pyparsing==3.1.1
+      - pyqt5-sip==12.11.0
+      - python-louvain==0.16
+      - scikit-learn==1.3.0
+      - scipy==1.11.2
+      - sympy==1.12
+      - threadpoolctl==3.2.0
+      - torch==2.0.1+cu117
+      - torch-cluster==1.6.1+pt20cu117
+      - torch-geometric==2.3.1
+      - torch-scatter==2.1.1+pt20cu117
```

Comment:
```suggestion
      - torch==2.0.1
      - torch-cluster==1.6.1
      - torch-geometric==2.3.1
      - torch-scatter==2.1.1
```

Reply:
It looks like this issue has to do with `torch-cluster` expecting an existing installation of `torch`. This means we won't be able to use a environment .yml file for installing `torch-cluster`.

I'll be adding a subsection in the repo's root README.md that will provide new installation instructions. (manual pip/conda installs, post-installation of torch)